tests/unit/amo/reducers/test_collections.js

```diff
@@ -801,15 +795,6 @@ describe(__filename, () => {
         fetchCurrentCollectionPage(partialParams);
       }).toThrow('username is required');
     });
-
-    it('throws an error when page is missing', () => {
```

Comment:
This test case was removed because the reducer method no longer expects a `page` parameter.